src/tasks/kube.ts

```diff
@@ -92,12 +106,21 @@ export class KubeTasks {
           const taskTitle = task.title
           const iterations = this.kubeHelper.podReadyTimeout / this.interval
           for (let i = 1; i <= iterations; i++) {
-            const failedState = await this.getFailedWaitingState(namespace, selector, 'Running')
-            if (failedState) {
+            // check cheCluster status
+            const cheClusterFailState = await this.getCheClusterFailState(namespace)
+
+            const failedState = await this.getContainerFailState(namespace, selector, 'Running')
+            if (cheClusterFailState || failedState) {
```

Comment:
I would suggest to fail only on operator failure and silently proceed to further check/recheck.